### ISSUE TITLE
Paper #1 Extension: Non-Detection Day Analysis (28.5% Miss Rate Defense)

### ISSUE THREAD

**Issue by iAmGiG** [iAmGiG/gex-llm-patterns#141]
# Paper #1 Extension: Non-Detection Day Analysis

**Issue Type**: Journal Revision Enhancement
**Priority**: P2 (Post-Workshop, Journal Submission)
**Effort**: Medium (2-3 days analysis + writing)
**Impact**: High (Strengthens "uniform regime" defense)

---

## Research Question

**What characteristics distinguish the 71 non-detection days (28.5%) from 171 detection days (71.5%)?**

This analysis directly addresses the "broken clock" criticism identified in external review.

---

### The Criticism (External Review)

**Quote from MC Review:**
> "Critical Weakness: The 'Uniform Regime' Problem. You explicitly state in Table VI that 100% of your sample days (242/242) were in a Negative Gamma regime (<-$2B). The Risk: A skeptic will argue the LLM didn't 'detect' a pattern; it simply learned (or guessed) the base rate of the 2024 environment. If the answer was 'Negative Gamma' 100% of the time, a 'broken clock' model guessing 'Negative Gamma' every time would have 100% accuracy."

> "The Defense: You need to vigorously defend why the 28.5% miss rate (the days it didn't detect the pattern) proves it wasn't just guessing. We need to look closely at what was different about those non-detection days."

### Current Defense (Already in Journal Version)

**File:** `docs/papers/paper1/journal_version/05_Results.tex` (lines 188-190)

> "The single-regime environment provides a particularly rigorous test of LLM structural reasoning: unlike traditional pattern recognition that exploits regime variation, our model must identify the underlying constraint mechanism within a uniform environment."

> "The 69.4% detection rate across 242 structurally identical days, combined with significant Granger causality, validates that LLMs recognize forced hedging constraints through mechanism-based reasoning rather than memorizing regime-switching patterns."

**Gap:** We state the 28.5% miss rate proves selectivity, but don't analyze WHAT makes non-detection days different.

---

### Issue #90 (CLOSED): Prompt Bias - 100% Detection Artifact
**Resolution:** Implemented unbiased obfuscation testing
- Detection dropped from 100% → 71.5% (proving selectivity)
- 28.5% miss rate IS the defense mechanism

### Issue #113 (CLOSED): Dealer Perspective Citations
**Relevance:** Established GEX limitations and robustness framework
- GEX measurement variations acknowledged
- Validation through forward returns (91.2% accuracy)

### Errata Document
**File:** `docs/papers/paper1/paper1-errata.md`
- Documents Issue #113 resolution
- Explains robustness mechanism
- **Does NOT address non-detection day characteristics** (this issue fills that gap)

---

### H1: Signal Strength (GEX Magnitude)
**Prediction:** Non-detection days have lower absolute GEX magnitude
- Detection days: Mean GEX ~ -$20B+
- Non-detection days: Mean GEX ~ -$5B to -$15B (weaker signal)

### H2: Signal Volatility (Intraday GEX Fluctuation)
**Prediction:** Non-detection days have higher intraday GEX changes
- Detection: Stable negative GEX throughout day
- Non-detection: GEX flipping or high variance (ambiguous signal)

### H3: Market Context (Volume, OI, Liquidity)
**Prediction:** Non-detection days have lower market activity
- Low volume/OI → weaker options market signal
- Low liquidity → noisier GEX estimates

### H4: Temporal Clustering
**Prediction:** Non-detection days cluster in specific periods
- Low-conviction regime transitions
- Holiday/low-activity periods
- Market microstructure shifts (e.g., OPEX weeks)

---

### Phase 1: Descriptive Statistics
Compare detection vs non-detection days across:

```python
# Group days by detection status
detection_days = validation_results[validation_results['detected'] == True]
non_detection_days = validation_results[validation_results['detected'] == False]

# Compare metrics
metrics = [
    'net_gex',
    'abs_gex',
    'gex_std',  # intraday volatility
    'volume',
    'open_interest',
    'forward_return',
    'realized_vol'
]

comparison = pd.DataFrame({
    'Detection Days': detection_days[metrics].mean(),
    'Non-Detection Days': non_detection_days[metrics].mean(),
    'T-Stat': ttest_ind(detection_days[metrics], non_detection_days[metrics])[0],
    'P-Value': ttest_ind(detection_days[metrics], non_detection_days[metrics])[1]
})
```

### Phase 2: Temporal Analysis
```python
# Check for clustering
non_detection_dates = non_detection_days['date'].values
date_gaps = np.diff(non_detection_dates)

# Visualize temporal distribution
plt.scatter(detection_days['date'], detection_days['net_gex'], c='green', label='Detection')
plt.scatter(non_detection_days['date'], non_detection_days['net_gex'], c='red', label='Non-Detection')
```

### Phase 3: Threshold Analysis
```python
# Find GEX threshold where detection probability changes
from scipy.stats import logistic

# Logistic regression: P(detection) ~ GEX magnitude
model = logistic.fit(abs_gex, detected)

# Identify cutoff point (e.g., 50% detection probability)
threshold_50pct = model.inverse_predict(0.5)
```

### Phase 4: Predictive Model
```python
# Can we predict non-detection based on day characteristics?
from sklearn.ensemble import RandomForestClassifier

features = ['abs_gex', 'gex_std', 'volume', 'oi', 'vix']
X = validation_results[features]
y = validation_results['detected']

rf = RandomForestClassifier().fit(X, y)
feature_importance = rf.feature_importances_
```

---

### Result 1: GEX Magnitude Threshold
**Expected Finding:**
- Detection days: Mean GEX = -$22.5B
- Non-detection days: Mean GEX = -$12.3B
- Threshold: Detection probability >70% when |GEX| > $15B

**Interpretation:** LLM requires minimum signal strength to confidently detect pattern.

### Result 2: Signal Clarity
**Expected Finding:**
- Detection days: Low intraday GEX volatility (stable signal)
- Non-detection days: High intraday GEX variance (noisy signal)

**Interpretation:** LLM discriminates between clear vs ambiguous constraints.

### Result 3: Not Random
**Expected Finding:**
- Non-detection days NOT uniformly distributed across year
- Clusters during: low-volume periods, OPEX transitions, regime uncertainty

**Interpretation:** Miss rate reflects genuine signal quality assessment, not random guessing.

---

### 1. Analysis Script
**Location:** `scripts/analysis/analyze_non_detection_days.py`

### 2. Results Report
**Location:** `reports/paper1/non_detection_analysis.md`

**Contents:**
- Summary statistics table (detection vs non-detection)
- Temporal distribution visualization
- Threshold analysis (GEX magnitude cutoffs)
- Predictive model feature importance
- Interpretation and conclusions

### 3. Journal Appendix Section
**Location:** `docs/papers/paper1/journal_version/08_Appendix.tex` (new file)

**Section:** "Appendix A: Non-Detection Day Characteristics"

**Content (~1-2 pages):**
- Table: Detection vs Non-Detection comparison
- Figure: Temporal distribution of non-detection days
- Figure: GEX magnitude threshold analysis
- Narrative: Why 28.5% miss rate proves selectivity, not base rate guessing

### 4. Updated Main Text
**Location:** `docs/papers/paper1/journal_version/05_Results.tex`

**Addition (after line 190):**
> "Detailed analysis of non-detection day characteristics (Appendix A) reveals that missed detections occur systematically when GEX magnitude falls below $15B (weak signal) or when intraday GEX volatility exceeds 30% (ambiguous signal). This demonstrates LLM selectivity rather than base rate guessing: the model requires minimum signal clarity to confidently identify constraints, with non-detections reflecting genuine signal quality assessment rather than random misses."

---

## Success Criteria

- [ ] Identify 2-3 statistically significant differences between detection/non-detection days (p < 0.05)
- [ ] Demonstrate non-detections are NOT randomly distributed (temporal clustering or threshold effects)
- [ ] Show predictive model can classify detection probability from day characteristics (AUC > 0.65)
- [ ] Write 1-2 page appendix suitable for journal submission
- [ ] Update main Results section with summary finding
- [ ] Strengthen defense against "broken clock" criticism

---

## Timeline

**Phase 1 (Analysis):** 1-2 days
- Run statistical comparisons
- Generate visualizations
- Identify key differentiators

**Phase 2 (Writing):** 1 day
- Create appendix section
- Update Results section
- Generate final report

**Phase 3 (Review):** 0.5 days
- Advisor review
- Incorporate feedback

**Total:** 2.5-3.5 days

---

## Dependencies

**Data Required:**
- ✅ Validation results (exists)
- ✅ Daily GEX data (exists)
- ✅ Detection status (exists)

**Code Required:**
- ✅ Validation framework (exists)
- ✅ Analysis utilities (exists)
- 🆕 New script (to create)

**No blockers - can execute immediately after Paper #1 workshop submission.**

---

### Strengthens Methodology Defense
- Proves 28.5% miss rate reflects signal discrimination, not randomness
- Demonstrates LLM threshold-based reasoning (minimum signal clarity required)
- Validates selectivity claims empirically

### Preempts Reviewer Criticism
- Directly addresses "broken clock" / base rate concern
- Turns potential weakness (uniform regime) into strength (rigorous test)
- Shows 100% negative gamma makes detection HARDER, not easier

### Publication Enhancement
- Adds methodological rigor expected in top journals
- Demonstrates thorough analysis of validation results
- Suitable for Journal of Finance, RFS, Management Science

---

## References

**Related GitHub Issues:**
- #90: Prompt Bias - 100% Detection Artifact (CLOSED)
- #113: Dealer Perspective Citations (CLOSED)

**Related Documentation:**
- `docs/papers/paper1/paper1-errata.md`: Issue #113 resolution
- `docs/papers/paper1/journal_version/README.md`: Journal expansion guidance

**External Review:**
- MC Review feedback (2025-11-22): "Uniform regime" criticism

---

**Priority:** P2 (Post-Workshop, Journal Revision)
**Status:** Backlog (activate after IEEE Big Data 2025 workshop submission)
**Related Paper:** Paper #1 - LLM Structural Reasoning via Obfuscation Testing
